paz/models/detection/model_util.py

```diff
@@ -0,0 +1,268 @@
+################################################
+### The functions in this file are direct    ###
+### copy of the ones present in the original ###
+### Frustum PointNet work                    ###
+################################################
+
+
+import numpy as np
+import tensorflow as tf
+import os
+import sys
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(BASE_DIR)
+
+# -----------------
+# Global Constants
+# -----------------
+
+NUM_HEADING_BIN = 12
+NUM_SIZE_CLUSTER = 8  # one cluster for each type
+NUM_OBJECT_POINT = 512

```

Comment:
this should be inputs to the model rather than hardcoded values found in another file